lib/core/kotekanMode.cpp

```diff
@@ -120,9 +120,15 @@ void kotekanMode::start_stages() {
         INFO_NON_OO("Starting kotekan_stage: {:s}...", stage.first);
         stage.second->start();
     }
+#if defined(CPU_MONITOR) && !defined(MAC_OSX)
```

Comment:
I'm going to suggest that we enable/disable the `CpuMonitor` via the configuration file instead of a `cmake` option.  There is no performance penalty to just always compiling `CpuMonitor` into the code, making it easier to enable for an existing binary later.   I'm going to propose the following config structure:
```
cpu_monitor:
    enabled: [bool]  # default false
    cpu_affinity: [array] # use the configuration scope to allow this to default to the "global" `cpu_affinity` value set in the config. 
```
See the `restServer::set_server_affinity()` for an example of how this might be done. 
The `#if !defined(MAC_OSX)` for MacOS builds can stay as is.